gettsim/parameters/eink_st_abzuege.yaml

```diff
@@ -535,6 +535,18 @@ behinderten_pauschbetrag:
     75: 1060
     85: 1230
     95: 1420
+  2020-01-01:
+    reference: Art. 1 G. v. 09.12.2020 BGBL. I  S. 2770.
+    20: 384
+    30: 620
+    40: 860
+    50: 1140
+    60: 1440
+    70: 1780
+    80: 2120
+    90: 2460
+    100: 2840
+    note: Gesetz gilt für das komplette Jahr 2020.
```

Comment:
Are you sure 2020? Because:

> „(33c) Die §§ 33 und 33b in der Fassung des Ar-
> tikels 1 des Gesetzes vom 9. Dezember 2020      
> (BGBl. I S. 2770) sind erstmals für den Veranla-    
> gungszeitraum 2021 anzuwenden.“

Reply:
Missed that one. Corrected now